app/grandchallenge/components/views.py

```diff
@@ -270,6 +276,29 @@ def get_context_data(self, *args, **kwargs):
         return context
 
 
+class CIVSetDetail(
+    LoginRequiredMixin,
+    ObjectPermissionRequiredMixin,
+    DetailView,
+):
+    model = None
+    permission_required = None
+    raise_exception = True
+    login_url = reverse_lazy("account_login")
+    template_name = "components/civ_set_detail.html"
+
+    @cached_property
+    def base_object(self):
+        return NotImplementedError
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+
+        context.update(dict(base_object=self.base_object))

```

Comment:
Using a literal here is more in line with what we do in the rest of the codebase. Did you have a specific reason for using the `dict()` constructor instead? 
```suggestion
        context.update({"base_object": self.base_object})

```

Reply:
I like the way the IDE's style the `dict()` constructor more than having `"argument"` everywhere. No longer applicable though, since this was refactored away.